components/prospects/ProfileSettings.tsx

```diff
@@ -0,0 +1,122 @@
+import * as React from 'react';
+import Menu from '@mui/material/Menu';
+import MenuItem from '@mui/material/MenuItem';
+import MoreVertIcon from '@mui/icons-material/MoreVert';
+import IconButton from '@mui/material/IconButton';
+import Router from 'next/router';
+import { Button, Dialog, DialogActions, DialogContent, DialogContentText, DialogTitle } from '@mui/material';
+
+export default function ProfileSettings({ id, name }) {
```

Comment:
It's important to keep the consistence of the code in the entire project, so only declare the function here, and then at the end of the file, make it `export default`. You can look at the other "component" files to see what I mean.